src/components/embedWrapper.tsx

```diff
@@ -332,6 +383,12 @@ const getSourceDetailsForEmbed = (embed: Embed): SourceDetails => {
 	}
 };
 
+const isblockedEditionsEmbed = (embed: Embed): boolean =>
+	embed.kind === EmbedKind.TikTok || embed.kind === EmbedKind.Instagram;
+
+const renderOverlay = (embed: Embed, editions: boolean): boolean =>
+	editions && !isblockedEditionsEmbed(embed);
```

Comment:
Will this return `false` whenever the product is _not_ Editions, i.e. won't it always return `false` on the live apps? Does this mean the overlay will never render on live apps?

Reply:
Yeah good point.